File: packages/rust/src/model/arena.rs
#[derive(Debug, Serialize, Deserialize, Clone)]
pub struct Arena {
    pub width: f64,
    pub height: f64,
    pub depth: f64,
    pub bottom_radius: f64,
    pub top_radius: f64,
    pub corner_radius: f64,
    pub goal_top_radius: f64,
    pub goal_width: f64,
    pub goal_height: f64,
    pub goal_depth: f64,
    pub goal_side_radius: f64,
}

File: packages/javascript/model/Arena.js
class Arena {
  constructor({
    width,
    height,
    depth,
    bottom_radius,
    top_radius,
    corner_radius,
    goal_top_radius,
    goal_width,
    goal_height,
    goal_depth,
    goal_side_radius
  }) {
    this.width = width;
    this.height = height;
    this.depth = depth;
    this.bottom_radius = bottom_radius;
    this.top_radius = top_radius;
    this.corner_radius = corner_radius;
    this.goal_top_radius = goal_top_radius;
    this.goal_width = goal_width;
    this.goal_height = goal_height;
    this.goal_depth = goal_depth;
    this.goal_side_radius = goal_side_radius;
  }
}
module.exports = Arena;
